samples/drivers/watchdog/Makefile

```diff
@@ -1,4 +1,12 @@
 BOARD ?= quark_d2000_crb
-CONF_FILE = prj.conf
+
+ifeq ($(BOARD),96b_carbon)
+CONF_FILE ?= prj_$(BOARD).conf
+else
+ifeq ($(BOARD),nucleo_f334r8)
+CONF_FILE ?= prj_$(BOARD).conf
```

Comment:
@erwango It seems that at this level of Makefile SoC series related if clauses don't have impact on prj_x.conf file being merged. Making the merging depended on the board seems to work only.